pom.xml

```diff
@@ -72,6 +72,11 @@
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>okhttp-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>1.49.0.59.vc32561e77cf3</version>
+    </dependency>
```

Comment:
The new extension seems to be optional, so this dependency should be as well, no?